scripts/setup-local-dev.sh

```diff
@@ -11,18 +11,17 @@ export ETHEREUM_WS=http://127.0.0.1:8546;
 
 export ETHEREUM_PRIVATE_KEY="fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a";
 
-SHOULD_SETUP_MARKETS=false;
-
-set +e;
-GETH_STATE=$(docker inspect -f {{.State.Running}} geth-node);
-if [[ "$GETH_STATE" != "true" ]]; then
-  echo "geth-node: ";
-  docker run -it -d --rm --name geth-node -p 8545:8545 -p 8546:8546 augurproject/dev-node-geth:latest;
-  SHOULD_SETUP_MARKETS=true;
-else
-  echo "geth-node: already running";
+CONTAINER_NAME="geth-node";
+DOCKER_IMAGE="augurproject/dev-pop-geth:core-$(npm view augur-core version)";
```

Comment:
dev-pop-geth:latest would work. There might be a chance that augur.js updates to use augur-core version that doesn't have corresponding docker image created. possible since creating the pop docker image is a manual process. still recommend :latest

Reply:
I did this to deal with old branches. Unsure how often this will happen.